### ISSUE TITLE
In Uchiwa 0.7.1-1 stash calls are not working

### ISSUE THREAD

**Issue by xyntrix** [sensu/uchiwa#316]
We run Uchiwa as such:

https://window.location/   will redirect via haproxy to Uchiwa, fronted with lighttpd+ldap for authentication
https://window.location/api/  will redirect via haproxy to Sensu's API, also fronted with lighttpd+ldap for authentication

Uchiwa is configured on the backend to connect to api URL 
http://window.location:4567  .. which connects via haproxy directly to the Sensu API and bypasses authentication.. and only the Uchiwa host is allowed to connect to that vip:port.

When I attempt to Silence an Alarm in Uchiwa 0.7.1-1, it attempts to fire the request in the browser to  https://window.location/post_stash, which appears to internally send data to the api http://window.location:4567 (which returns a 500 error).   

TL;DR:   stash commands are returning a 500 error, was working in 0.4.0

**Comment by palourde** [comment]
Hi @xyntrix 

Could you provide me the logs from Uchiwa and your browser console when you attempt to create a stash?

Thanks

**Comment by mbobakov** [comment]
Same issue on 0.8.1 version.
Config: 
{
  "Sensu": [
    {
      "Name": "eu",
      "Host": "SENSUAPISERVER",
      "Port": 4567,
      "Ssl": false,
      "Insecure": false,
      "URL": "http://SENSUAPISERVER:4567",
      "User": "*****",
      "Path": "",
      "Pass": "*****",
      "Timeout": 10
    }
  ],
  "Uchiwa": {
    "Host": "0.0.0.0",
    "Port": 3000,
    "Refresh": 5,
    "Pass": "*****",
    "User": "*****",
    "Db": {
      "Driver": "",
      "Scheme": "*****"
    },
    "Auth": ""
  }
}


And I try for example remove stash for the host and .... 404-error:
General properties of request in Chrome: 

Remote Address:10.148.4.18:3000
Request URL:http://SENSUAPISERVER:3000/stashes/delete
Request Method:POST
Status Code:404 Not Found

Uchiwa requesting port 3000 instead 4567 !

**Comment by palourde** [comment]
@mbobakov Make sure to restart the Uchiwa process after upgrading.

**Comment by warmfusion** [comment]
Had this same problem after updating Sensu servers to 0.18.0  and using Uchiwa 8.1 - Restarted processes and things fixed up nicely.

**Comment by mbobakov** [comment]
Oh..really. Restart is needed. In my case Puppet didn't restart service. Thank you

**Comment by palourde** [comment]
Hi @xyntrix 

Were you able to fix this issue by restarting the Uchiwa process?

**Comment by palourde** [comment]
Closing due to inactivity. This can be reopened if necessary.

**Comment by joelmoss** [comment]
I get the same problem with Uchiwa 0.8.1 and Sensu 0.19.1. Even after restarting api, client and server I still get 404's on POST /stash

```
"The stash was not created. 404 page not found\n"
```

**Comment by xyntrix** [comment]
Sorry for not getting back sooner!  Yes, restarting Uchiwa fixed the issue.
